src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -16,9 +16,13 @@
 
 import codeu.model.data.Message;
 import codeu.model.store.persistence.PersistentStorageAgent;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.UUID;
+
+import java.util.*;
```

Comment:
It might be clearer to just import only the packages that you use, instead of all of util.